modules/nix-bitcoin-services.nix

```diff
@@ -51,6 +51,15 @@ let self = {
     '';
   };
 
+  getAnnounceAddressCmd = mkOption {
```

Comment:
This seems to be the wrong commit for this option because it's already inherited by the spark module in the previous commit.

Reply:
I'll fix this in the final squash.